pkg/spec/backup.go

```diff
@@ -40,7 +40,4 @@ type BackupPolicy struct {
 	// StorageType specifies the type of storage device to store backup files.
 	// If it's not set by user, the default is "PersistentVolume".
 	StorageType BackupStorageType `json:"storageType"`
-	// CleanupOnClusterDelete specified whether we want to cleanup the backup data if cluster is deleted.
-	// By default, operator will keep the backup data.
-	CleanupOnClusterDelete bool `json:"cleanupOnClusterDelete"`
```

Comment:
i think we should still keep this. this is convenient for people who wants to setup a temp etcd for testing/developing purpose.